Lesson01/LMS/MainWindow.xaml.cs

```diff
@@ -23,7 +23,8 @@ public partial class MainWindow : Window
         public MainWindow()
         {
             InitializeComponent();
-// comment
+
+            int a = "hello";
```

Comment:
It is not possible to assign "hello" to integer type, please change it data type to string.